docs/tech_specs/monitoring_data_consumption.md

```diff
@@ -0,0 +1,61 @@
+
+## Goals 
+There are two main goals in the scope of this tech spec because there are two separate ways we will be getting the data for message logs and sign uptime, respectively.
+
+  1. Set up a recurring process that will request ARINC’s headend server on a nightly basis for a file containing message logs over the last 24 hours and then stream these logs into Splunk.
+  2. Set up an endpoint for ARINC to send requests to CTD informing whether a given sign is up or down at some frequent interval (something like every 30 seconds – 1 Min)
+
+The order in which these goals are accomplished is arbitrary, or they could be worked on in parallel.
+
+## Proposed solution
+### Part 1: Message Logs 
+
+To give more context on the reasoning behind the solution proposed, we should note that Realtime Signs runs on `opstech3` which has historically been prone to run out of memory and experience other performance issues. It's also worth noting that opstech3 is intended to be a DB server and not meant to host applications like Realtime Signs. Therefore, we should try to keep the footprint of whatever new logic/features we add to Realtime Signs as minimal. Also, it is to our advantage to host processing-heavy logic in AWS since that allows us to scale resources if we need to. We don't have that ability on opstech3.
+
+To give a rough idea of the required resources for processing the message logs, the CSV files may be up to ~200MB which is based on the number of messages we send to ARINC over the course of a day. While this ultimately isn’t a ton of data, we should note that this is only counting visual messages. We also send audio messages, but they are not able to be logged yet. If/when those are added, the number of rows in the CSV would at least double.
+
+As a list of steps, the solution would involve the following in sequence: 
+
+  1. ARINC builds out an application that filters out unneeded logs and compiles the data that we outlined in our specification. With CTD having improved the usability of the `uid` field that we send in our messages to the ARINC headend server, this is achievable now.
+
+  2. ARINC sets up an endpoint for CTD to hit once per day around 2AM to receive a file containing the compiled logs. It would probably be good if the file is a zip to keep the size of data being transferred between servers small.
+
+  3. CTD sets up a recurring job which calls ARINC’s endpoint nightly to receive the file and dump it into an S3 bucket. Having the logs stored in S3 also makes it convenient if we ever need to dig up past log files for some reason.
```

Comment:
This seems good. You could potentially use the new data platform S3 buckets, which we're planning to use for data such as this. (I'd still recommend starting with Splunk for the visualization)